web/src/js/ducks/ui/keyboard.js

```diff
@@ -124,9 +119,12 @@ export function onKeyDown(e) {
                 break
             }
 
-
+            // ignore all other keys
             default:
                 return
         }
+        
+        // prevent key press if it was handled
+        e.preventDefault()
```

Comment:
Oh I see the problem here. `e` is not defined in this scope as lambda functions create a new scope IIRC.

Okay, dunno how to solve this. Pass `e` into the function or just remove this lambda function somehow and get the other variables somewhere else?